src/Components/Users/ManageUsers.tsx

```diff
@@ -439,7 +439,7 @@ export default function ManageUsers() {
         setOpen={setExpandFacilityList}
         slideFrom="right"
         title="Facilities"
-        dialogClass="w-[350px]"
+        dialogClass="w-[350px] p-6"
```

Comment:
@AbhishekPSingh07 adding padding is not the right solution. Making it responsive is the right solution.

```suggestion
        dialogClass="md:w-[400px]"
```